pmda/test/test_util.py

```diff
@@ -15,8 +15,9 @@
 import time
 import numpy as np
 from numpy.testing import assert_almost_equal, assert_equal
+from functools import reduce
```

Comment:
I recommend not importing functions in the top level name space because it gets hard to figure out where they are from, especially if they're not from the package itself. I would
```python
import functools
...
z = functools.reduce(X, Y)
```
In this way it is immediately clear, where `reduce()` comes from.